src/screens/home/HomeScreen.tsx

```diff
@@ -218,10 +219,11 @@ export const HomeScreen = () => {
   const startAndUpdate = useCallback(async () => {
     if (userStopped) return;
     const success = await startExposureNotificationService();
+    checkForExposures();
```

Comment:
This means the outbreak checks won't happen if the user has turned the app off. Good call.

Reply:
Can you put this inside an `if (QR_ENABLED) {...}` check? Might cause an error if the feature isn't turned on.